assets/src/edit-story/app/media/utils/createResource.js

```diff
@@ -55,6 +56,7 @@ import getTypeFromMime from './getTypeFromMime';
 function createResource({
   type,
   mimeType,
+  uploadDate,
```

Comment:
What do you think of 'createTime' as an alternative? It would be particularly useful if we decide to re-use this shape for the Unsplash integration.